### PR TITLE
Quantify estimated uplift of static tag loading in audit failure value

### PR DESCRIPTION
**Pull request by warrengm**
Estimates the expected uplift (w.r.t. tag load time) of loading gpt statically and displays the result in the audit's failure display title

### CODE REVIEW

File: lighthouse-plugin-ad-speed-insights/messages/en-US.json [hunk 1]
```diff
@@ -89,7 +89,8 @@
     "static-ad-tags": {
       "title": "GPT tag is loaded statically",
       "failureTitle": "Load GPT tag statically",
-      "description": "Tags loaded dynamically are not visible to the browser preloader. Consider using a static tag or `<link rel=\"preload\">` so the browser may load GPT sooner. [Learn more](https://ad-speed-insights.appspot.com/#dynamic-tag)."
+      "description": "Tags loaded dynamically are not visible to the browser preloader. Consider using a static tag or `<link rel=\"preload\">` so the browser may load GPT sooner. [Learn more](https://ad-speed-insights.appspot.com/#dynamic-tag).",
+      "failureDisplayValue": "%s sec estimated tag load speed-up"
```
Comment (jburger424) [inline lighthouse-plugin-ad-speed-insights/messages/en-US.json]: sec => s for consistency.

File: lighthouse-plugin-ad-speed-insights/audits/static-ad-tags.js [hunk 2]
```diff
@@ -99,8 +119,19 @@ class StaticAdTags extends Audit {
     const dynamicReq = tagReqs.find((r) => !isStaticRequest(r));
     const table = dynamicReq ? getDetailsTable(dynamicReq) : [];
 
+    const failed = numStatic < numTags;
+    let displayValue;
```
Comment (jburger424) [inline lighthouse-plugin-ad-speed-insights/audits/static-ad-tags.js]: Does this work fine as undefined rather than an empty string?

Comment (warrengm) [reply lighthouse-plugin-ad-speed-insights/audits/static-ad-tags.js]: Seems like it, but changed it to the empty string to be safe

File: lighthouse-plugin-ad-speed-insights/messages/en-US.json (hunk 1 above)
Comment (warrengm) [reply lighthouse-plugin-ad-speed-insights/messages/en-US.json]: Done

**Pull request merged**